butterfree/transform/features/timestamp_feature.py

```diff
@@ -67,7 +67,7 @@ def transform(self, dataframe: DataFrame) -> DataFrame:
 
         if self.from_ms:
             dataframe = dataframe.withColumn(
-                column_name, from_unixtime(dataframe[column_name] / 1000.0)
+                column_name, to_timestamp(dataframe[column_name] / 1000)
```

Comment:
You could simplify this method with:

```python
ts_column = dataframe[column_name]
if self.from_ms:
    ts_column = ts_column / 1000

return dataframe.withColumn(column_name, to_timestamp(ts_column, self.mask)
```
since, by default, `self.mask` is None and both transformations can work together :)

Reply:
I thought about this, but wasn't sure what would happen when `self.mask` was null... but checking documentation it's the default behavior.